added a newer Greek[el_GR] dictionary

gzougianos opened the pull request:
[Previous el_GR.dict](https://github.com/languagetool-org/languagetool/blob/master/languagetool-language-modules/el/src/main/resources/org/languagetool/resource/el/hunspell/el_GR.dict) was uploaded 6 years ago (Aug 8 2013). I created a new one based on [elspel.gr](http://www.elspell.gr/myspell) latest dictionary version.

danielnaber commented:
Thanks! Could you also update the `README_el_GR.txt` in the same directory? Also, it would be helpful to document the exact steps for creating this file.

gzougianos commented:
I have updated the readme file with everything I thought it was needed. I also added the link of [the dictionary builder gui I have created](https://github.com/gzougianos/languagetool-dictionarybuilder-gui) (in order to build the dictionaries easier).
